### ISSUE TITLE
@return is not styled properly if there are multiple return statements

### ISSUE THREAD

**Issue by fulldecent** [juanfranblanco/vscode-solidity#243]
Test case:

<img width="431" alt="Screen Shot 2021-05-14 at 11 02 47" src="https://user-images.githubusercontent.com/382183/118289999-ee8fc300-b4a3-11eb-9bba-5c61850be1ff.png">

The c and d should be highlighted.

**Comment by juanfranblanco** [comment]
https://github.com/juanfranblanco/vscode-solidity/blob/master/syntaxes/solidity.json#L110-L124

This may need to match the regex for params

**Comment by fulldecent** [comment]
It's a little more complicated. The `@return` only gets a name parameter if the return value has a name in the `returns(...)`block.

This is distinct from `@param` where it always has a name.

**Comment by juanfranblanco** [comment]
Yeah, so I guess there is not a simple true way to do this, as you may have also anonymous return parameters.
